<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>765</width>
      <height>741</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Current folder: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="currentFolderLine"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>git flow</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="featureCheck">
         <property name="text">
          <string>feature</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="hotFiCheck">
         <property name="text">
          <string>hotFix</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="startCheck">
         <property name="text">
          <string>start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="finishCheck">
         <property name="text">
          <string>finish</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="gitName">
         <property name="toolTip">
          <string>insert feature name here</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>execute</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>git</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="addCheck_2">
         <property name="text">
          <string>add *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="commitCheck_2">
         <property name="text">
          <string>commit -m</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="pushCheck">
         <property name="text">
          <string>push</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="gitArgument">
         <property name="toolTip">
          <string>insert feature name here</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="execute2button">
         <property name="text">
          <string>execute</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>custom command</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="customCommandLabel">
         <property name="toolTip">
          <string>insert feature name here</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="executeCustomCommand">
         <property name="text">
          <string>execute</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="output">
       <property name="font">
        <font>
         <family>Monospace</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(170, 255, 0);</string>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>784</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionGit_project_root_path"/>
    <addaction name="actionGit_flow_executable"/>
   </widget>
   <widget class="QMenu" name="menuGit">
    <property name="title">
     <string>Git</string>
    </property>
    <addaction name="actionGit_branch"/>
    <addaction name="actionGit_flow_init"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuGit"/>
  </widget>
  <action name="actionGit_project_root_path">
   <property name="text">
    <string>Git project root path</string>
   </property>
  </action>
  <action name="actionGit_flow_executable">
   <property name="text">
    <string>Git flow executable</string>
   </property>
  </action>
  <action name="actionGit_branch">
   <property name="text">
    <string>Git branch</string>
   </property>
  </action>
  <action name="actionGit_flow_init">
   <property name="text">
    <string>Git flow init</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
